client/src/resources/lang/en_us.json

```diff
@@ -1180,6 +1188,20 @@
     "wiki.article.standard.noDeathCause.text":"No Death Cause is a game modifier that changes the game such that every grave does not have a death cause listed.",
     "wiki.article.standard.roleSetGraveKillers.title":"Role Set Grave Killers",
     "wiki.article.standard.roleSetGraveKillers.text":"Role Set Grave Killers is a game modifier that changes the game such that every grave killer which is a role is replaced with one of:\n* Syndicate;\n* Town;\n* Minions;\n* Neutral;\n* Fiends; or\n* Cult.\n\nOr, if the killer's role is not in one of these role sets, it is not changed.",
+    "wiki.article.standard.noDueProcess.title": "No Due Process",
+    "wiki.article.standard.noDueProcess.text": "No Due Process is a game modifier that changes the game such that every nominated player will be executed without judgement.",
+    "wiki.article.standard.twoThirdsMajority.title": "Two-thirds Majority",
+    "wiki.article.standard.twoThirdsMajority.text": "Two-thirds Majority is a game modifier that changes the game such that during nomination and judgement, a two-thirds majority is required, rather than a simple majority.",
+    "wiki.article.standard.noTrial.title": "No Trial",
+    "wiki.article.standard.noTrial.text": "No Trial is a game modifier that changes the game such that the trial phases Nomination, Testimony, and Judgement no longer happen.",
+    "wiki.article.standard.noWhispers.title": "No Whispers",
+    "wiki.article.standard.noWhispers.text": "No Whispers is a game modifier that changes the game such that no players can whisper each other.",
+    "wiki.article.standard.noBlockMessages.title": "No Block Messages",
+    "wiki.article.standard.noBlockMessages.text": "No Block Messages is a game modifier that changes the game such that block messages cannot be sent.",

```

Comment:
What happens when you try? What happens when you go to send your alibi?

Reply:
Button is greyed out

Reply:
It should say that. Id rather you remove this modifier altogether though